.github/workflows/build_vllm.yaml

```diff
@@ -38,10 +38,9 @@ jobs:
               "package_type": "manywheel",
               "build_name": "manywheel-py3_10-cuda12_8",
               "validation_runner": "linux.12xlarge.memory",
-              "installation": "pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu128",
+              "installation": "pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu128",
               "channel": "test",
               "upload_to_base_bucket": "no",
-              "stable_version": "2.9.0",
```

Comment:
we just need to upgrade this line as this flow is to build vLLM against the pinned torch version used in the `pyproject.toml`